### PR TITLE
Target minimal API Sets version for AdvApi32

### PR DESCRIPTION
**Pull request by jmelosegui**
Work for #84

### CODE REVIEW

File: src/AdvApi32.Shared/AdvApi32.cs [hunk 1]
```diff
@@ -0,0 +1,649 @@
...
+        private const string api_ms_win_security_base_l1_2_0 = ApiSets.api_ms_win_security_base_l1_2_0;
+        private const string api_ms_win_service_winsvc_l1_2_0 = ApiSets.api_ms_win_service_winsvc_l1_2_0;
+#else
+        private const string api_ms_win_service_core_l1_1_1 = nameof(AdvApi32);
+        private const string api_ms_win_service_management_l1_1_0 = nameof(AdvApi32);
+        private const string api_ms_win_service_management_l2_1_0 = nameof(AdvApi32);
+        private const string api_ms_win_core_processthreads_l1_1_1 = nameof(AdvApi32);
+        private const string api_ms_win_security_base_l1_2_0 = nameof(AdvApi32);
+        private const string api_ms_win_service_winsvc_l1_2_0 = nameof(AdvApi32);
+#endif
```
Comment (AArnott) [inline src/AdvApi32.Shared/AdvApi32.cs:38]: Should you add this after the `#endif`?

``` csharp
#pragma warning restore SA1303 // Const field names must begin with upper-case letter
```

File: src/Kernel32.Shared/Kernel32.cs [hunk 2]
```diff
@@ -37,11 +42,6 @@ public static partial class Kernel32
 #pragma warning restore SA1303 // Const field names must begin with upper-case letter
 
         /// <summary>
-        /// Constant for invalid handle value
-        /// </summary>
-        public static readonly IntPtr INVALID_HANDLE_VALUE = new IntPtr(-1);
```
Comment (AArnott) [inline src/Kernel32.Shared/Kernel32.cs:42]: Did stylecop want this reordered? I thought all constants were supposed to appear before all static fields.

**Comment by AArnott** [comment]
Looks really good. Thanks.
Just fix the build break and comments, then feel free to merge.

File: src/Kernel32.Shared/Kernel32.cs (hunk 2 above)
Comment (jmelosegui) [inline src/Kernel32.Shared/Kernel32.cs:42]: Yes. Public before Private takes precedence.

**Comment by AArnott** [comment]
BTW, I found some value in the fact that the Kernel32 unit test projects had a desktop and profile92 version so that both APISets and kernel32.dll methods were exercised. Should we do that here as well?

**Comment by jmelosegui** [comment]
I would do it as soon as we have at least one unit test. My concern here is why we don't have unit tests.

Regarding windows services.
- PInvoke against windows services tests would involve create a service (and builds with elevated privileges)
- What else could we test here, the helper methods?

**Comment by AArnott** [comment]
Fair point on the tests.

**Pull request merged**